### ISSUE TITLE
Any Updates on this?

### ISSUE THREAD

**Issue by WyvrnRipsnarl** [InvoxiPlayGames/ReFused#2]
Is there been progress on this project, should we still hope to be able to play in coop again or getting some modes restablished?

**Comment by supersonic124** [comment]
its joever.....................................

**Comment by WyvrnRipsnarl** [comment]
I know the game is out of direct service but I had a feeling we get more developpment from Invoxi about reestablishing the coop features, guess that's a dead project too then...

**Comment by Spooky290** [comment]
Soo no updates on this

**Comment by polocatfan** [comment]
rip

**Comment by MrKizu** [comment]
Sounds like an abandoned project. Big sad really.

**Comment by BeardyD20** [comment]
Yes it's very sad. I've no idea how these things work, just a Fuser fan, but if anyone could do it for a fee, would that work? There is a present and legacy community that could be contacted to crowdfund. I'd be happy to contribute to ongoing server costs too.

**Comment by InvoxiPlayGames** [comment]
Unfortunately reversing actual real-time stuff like the Diamond Stage and co-op stuff is much different to how the rest of the server-side code of the game works. I do have packet captures and Fiddler logs *of the Diamond Stage* I'd be willing to share once I can suitably make sure there's nothing personal left over, but that's not only going to be a lot of work but also I don't know how much of it actually counts for co-op. (Also I only have Fiddler captures and Wireshark packet captures; I didn't think to capture the memory contents of the game, so if it uses some sort of Diffie-Hellman or similar key exchange then it's impossible to reverse from those captures. I didn't look into it.)

Sorry for "abandoning" the project- I lost most interest in FUSER and nobody else seemed to be offering to actually work on it or submitting pull request so it's had to lay dormant, and due to the UK's Online Safety Act I had to stop hosting my testing instance of the server regardless for legal reasons.

(As an aside, **I would heavily advise against getting donations or money involved with modding Harmonix games.** My previous offer for packet captures and Fiddler logs only stands as long as the effort is solely motivated out of passion rather than for money.)

**Comment by BeardyD20** [comment]
Thanks for the reply! Yeah, I wouldn't want to get anyone in any legal difficulty, don't risk that. I'd just love to get co-op back, it got me through the pandemic, and offering a bit of money or trying to help organise some is just about all I could offer, given my complete lack of technical expertise. I think it's great that people do these things for passion, but everyone's time is valuable, and if a bit of compensation helped, I just wanted to offer it anyway. Thanks for your efforts anyway.
